src/main.rs

```diff
@@ -1,3 +1,40 @@
-fn main() {
-    println!("Hello, world!");
+use teloxide::dispatching::dialogue::InMemStorage;
+use teloxide::prelude::*;
+use tg_bot::commands::{schema, State};
+use tg_bot::configuration::get_configuration;
+use tg_bot::startup::{get_connection_pool, Application};
+
+#[tokio::main]
+async fn main() -> std::io::Result<()> {
+    let configuration = get_configuration().expect("Failed to read configuration.");
+
+    let bot = Bot::new("TOKEN_BOT");
+
+    let webhook_url = format!("https://0000-000-000-000-00.ngrok-free.app/TOKEN_BOT")
```

Comment:
this literal should not be hardcoded

Reply:
I totally agree for security it is better to add in envs of course. In this form I did it more for clarity.